### PR TITLE
Increase sample rate

### PR DESCRIPTION
**Pull request by VictorCarri**
Added preliminary code which should go towards resolving issues #17 and #5.

### CODE REVIEW

File: cleansio/audio/Libs used [hunk 1]
```diff
@@ -0,0 +1,8 @@
+convert.py
+    AudioSegment (python)
```
Comment (PatrickDuncan) [inline cleansio/audio/Libs used]: Can you create a new page on the Wiki for this?

Comment (VictorCarri) [reply cleansio/audio/Libs used]: OK

Comment (PatrickDuncan) [reply cleansio/audio/Libs used]: Hey, I meant the library list from #17, not the sampling class for the wiki page. Also check out my PR

**Comment by PatrickDuncan** [comment]
Resolved by #28. #17 doesn't need a PR

**Pull request closed without merging**